cmd/veneur-emit/main.go

```diff
@@ -437,6 +515,30 @@ func sendStatsd(addr string, span *ssf.SSFSpan) error {
 	return nil
 }
 
+func validateFlagCombinations(passedFlags map[string]flag.Value) {
+	// Figure out which mode we're in
+	var mode EmitMode
+	mv, has := passedFlags["mode"]
+	if !has {
+		mode = MetricMode
```

Comment:
Took me a bit to parse this, lave a comment "Default to metric mode if we didn't get a mode" or something?